pex/pep_440.py

```diff
@@ -73,6 +73,12 @@ def __lt__(self, other):
             return NotImplemented
         return self.parsed_version < other.parsed_version
 
+    def __ge__(self, other):
```

Comment:
Unfortunately this `__ge__` and the one below in `PipVersionValue` are needed to satisfy MyPy despite using `@total_ordering` in both cases.